torch/nn/quantized/modules/linear.py

```diff
@@ -167,10 +163,8 @@ def __getstate__(self):
     # weight into its packed format for use by the FBGEMM ops.
     def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                               missing_keys, unexpected_keys, error_msgs):
-        self.set_weight(state_dict[prefix + 'weight'])
+        self.set_weight_bias(state_dict[prefix + 'weight'], state_dict[prefix + 'bias'])
```

Comment:
Similar comment, lets have individual functions for weight and bias

Reply:
The set_weight_bias function calls the prepack method, which needs both weight and bias as arguments. I don't think it will help to separate them out.